lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -836,29 +839,35 @@ public function getAddExtendedPropertySQL(
      * @link http://technet.microsoft.com/en-gb/library/ms178595%28v=sql.90%29.aspx
      *
      * @param string      $name       The name of the property to drop.
-     * @param string|null $level0Type The type of the object at level 0 the property belongs to.
-     * @param string|null $level0Name The name of the object at level 0 the property belongs to.
-     * @param string|null $level1Type The type of the object at level 1 the property belongs to.
-     * @param string|null $level1Name The name of the object at level 1 the property belongs to.
+     * @param string      $level0Type The type of the object at level 0 the property belongs to.
+     * @param string      $level0Name The name of the object at level 0 the property belongs to.
+     * @param string      $level1Type The type of the object at level 1 the property belongs to.
+     * @param string      $level1Name The name of the object at level 1 the property belongs to.
      * @param string|null $level2Type The type of the object at level 2 the property belongs to.
      * @param string|null $level2Name The name of the object at level 2 the property belongs to.
      *
      * @return string
      */
     public function getDropExtendedPropertySQL(
         $name,
-        $level0Type = null,
-        $level0Name = null,
-        $level1Type = null,
-        $level1Name = null,
-        $level2Type = null,
-        $level2Name = null
+        string $level0Type,
```

Comment:
This is a breaking change.

Reply:
Discussed in https://github.com/doctrine/dbal/pull/4360#discussion_r602897895 , not changing it in this PR.